src/Runner.Worker/ExecutionContext.cs

```diff
@@ -92,7 +98,7 @@ public interface IExecutionContext : IRunnerService
         void SetGitHubContext(string name, string value);
         void SetOutput(string name, string value, out string reference);
         void SetTimeout(TimeSpan? timeout);
-        void AddIssue(Issue issue, string message = null);
```

Comment:
`void AddIssue(Issue issue, string message = null, bool writeToLog = true);`
☝️  will this help reduce the amount of code changes since we don't need to change all existing callers?

Reply:
There are only 11 callers outside of L0 tests.  My usual rule-of-thumb for resorting to an optional param is around 200 pre-existing callers for an unexported method.

Reply:
10+ is a lot for the runner since it's a tiny codebase 😄